index.html

```diff
@@ -1793,7 +1801,7 @@ <h3>
               </p>
 
               <pre class="example highlight" title="Data vault batch operation request">
-  POST  /encrypted-data-vaults/z4sRgBJJLnYy HTTP/1.1
+  POST  /encrypted-data-vaults/z4sRgBJJLnYy/batch HTTP/1.1
```

Comment:
I'm wondering if the `/batch` part of the URL is even needed. 

Maybe a `POST /encrypted-data-vaults/z4sRgBJJLnYy` would be sufficient? (Since the payload is an array. (vs, if it was a regular EDV document, it'd be an object)).

Reply:
I agree @dmitrizagidulin - I just updated the PR to remove the `/batch`. (If anyone else feels differently, though - let us know :) )